functions.php

```diff
@@ -373,6 +374,36 @@ public function p4_save_page_type( $post_id ) {
 		wp_set_post_terms( $post_id, sanitize_text_field( $selected->slug ), 'p4-page-type' );
 	}
 
+	/**
+	 * Call on publish a post.
+	 */
+	public function send_mails_on_publish( $ID, $post ) {
+		$emails  = [];
+		$args    = [
+			'role' => 'editor'
+		];
+		$users   = get_users( $args );
+		if( $users ) {
+			foreach( $users as $user ) {
+				$emails[] = $user->user_email;
+			}
+		}
+
+		if( $emails ) {
+			$headers  = 'MIME-Version: 1.0' . "\r\n";
+			$headers .= 'Content-type: text/html; charset=iso-8859-1' . "\r\n";
+			$name     = $post->post_name;
+			$subject  = __( 'New post published', 'planet4-master-theme' );
+			$body     = sprintf( 'Hello Team, <br><br>' );
+			$body    .= sprintf( 'The post <strong> %s </strong> has been published. <br>', $name );
+			$body    .= sprintf( 'Post link : %s <br><br>', get_permalink( $post ) );
+			$body    .= sprintf( 'Thanks<br>' );
+			$body    .= sprintf( 'Greenpeace IT' );
```

Comment:
@sagarsdeshmukh Should these strings be translatable?

Reply:
Yes it is but I assume its just an initial basic version of mail content and in future, we will change this mail template  @koyan : Do you have any suggestion for this ?

Reply:
And should this test be in UTF instead of ISO8859?